src/main/java/duke/DeadlineTask.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.TextStyle;
+import java.util.Locale;
+
+/**
+ * Represents a deadline task with a deadline.
+ */
+public class DeadlineTask extends Task {
+    LocalDate deadline;
```

Comment:
Maybe set private and final for variables? Since you're not modifying it anymore

Reply:
Resolved.